builtin/providers/cloudflare/resource_cloudflare_record_migrate.go

```diff
@@ -0,0 +1,96 @@
+package cloudflare
+
+import (
+	"fmt"
+	"log"
+	"strconv"
+
+	"github.com/hashicorp/terraform/terraform"
+	// NOTE: Temporary until they merge my PR:
+	"github.com/mitchellh/cloudflare-go"
+)
+
+func resourceAwsCloudFlareRecordMigrateState(
```

Comment:
Small nit: Cloudflare isn't part of AWS, so I don't think there's a reason to include 'AWS' in the function name.